src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,104 @@
+package duke;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Scanner;
+import java.io.*;
```

Comment:
Avoid wildcard import statements for packages. Explicitly mention all the classes being used.